index.js

```diff
@@ -127,6 +131,7 @@ let scope = function(commonTags) {
     stop: stopAtlas,
     setDevMode: devMode,
     getDebugInfo: debugInfo,
+    analyzeId: analyzeId,
```

Comment:
I thought it could be useful in a more general way, but I wouldn't have a problem reducing the visibility. My main use case was for someone not necessarily using the mocks, and wanting to do the validation in some tests prior to calling the `.counter`, `.timer`, etc. functions.

Reply:
Sounds good to me.